src/collective/fingerpointing/tests/test_utils.py

```diff
@@ -13,15 +12,98 @@ def setUp(self):
         self.request = self.layer['request']
 
     def test_get_request_information(self):
+        from collective.fingerpointing import utils
```

Comment:
same here: what's the point on importing this all over the place?

Reply:
as said, as late as possible in tests to let it raise in the test and not at startup. Error logging and message in test output are better. Its a best practice in order to keep my developer life sane. See also imports in the tests.

Reply:
FYI @idgserpro